osquery/tables/events/windows/windows_events.cpp

```diff
@@ -0,0 +1,115 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <boost/property_tree/ptree.hpp>
+
+#include <osquery/config.h>
+#include <osquery/core.h>
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+
+#include <boost/property_tree/xml_parser.hpp>
+#include <osquery/core/conversions.h>
+#include <osquery/core/json.h>
+#include <osquery/core/windows/wmi.h>
+
+#include "osquery/events/windows/windows_event_log.h"
+#include "osquery/filesystem/fileops.h"
+
+namespace pt = boost::property_tree;
+
+namespace osquery {
+
+FLAG(uint64,
+     windows_events_expiry,
+     60 * 60 * 24 * 30, // Keep 30 days by default
```

Comment:
You could remove these and "inherit" the values from `events_expiry` and `events_max`.